app/views/main.py

```diff
@@ -22,21 +22,21 @@
 logger = logging.getLogger(__name__)
 
 
-root_blueprint = Blueprint('root', __name__)
```

Comment:
Give me a more appropriate name and I'll change it. My concern with the name "root" was it implies some level of access 'main' doesn't.